frontend/src/components/commons/BackToTop.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+import scrollToTop from "../../helpers/scrollToTop";
+
+const styles = {
+  displayedButton: {
+    display: "block",
+    position: "fixed",
+    bottom: "5%",
+    right: "3%"
+  }
+};
+
+class BackToTop extends Component {
+  state = {
+    buttonVisible: false
+  };
+
+  componentDidMount = () => {
+    window.addEventListener("scroll", this.handleOnScroll);
+  };
+
+  // handle onScroll event
+  handleOnScroll = () => {
+    const currentScroll = document.documentElement.scrollTop;
+    console.log(currentScroll);
```

Comment:
remove console.logs :)

Reply:
oups! Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/87/commits/1a37cc8815f328af23bc18fc4e37a42e6aa5ec4e